src/CloudTasksQueue.php

```diff
@@ -237,13 +246,18 @@ public function release(CloudTasksJob $job, int $delay = 0): void
 
         $payload = $job->getRawBody();
 
-        $options = ['delay' => $delay];
+        $options = ['delay' => $delay, 'job' => $job];
 
         $this->pushRaw($payload, $job->getQueue(), $options);
     }
 
-    public function getHandler(): string
+    /** @param string|object $job */
+    public function getHandler(mixed $job): string
     {
+        if ($job instanceof HasTaskHandlerUrl) {
+            return $job->taskHandlerUrl();
```

Comment:
Good point. I understand the need and use case, but is this something this package needs to take care of, or is this logic that can be put in the `taskHandlerUrl()` method of the job? Because I can imagine that jobs A and B will behave differently than jobs C and D, url wise (and environment) wise.

I do like the option to have multiple handlers in the config, but personally I would not hard-code these.  Other than a `'default'`.

Something like this:
```
'cloudtasks' => [
    'handler'               => env('STACKKIT_CLOUD_TASKS_HANDLER', ''),
    'service_account_email' => env('STACKKIT_CLOUD_TASKS_SERVICE_EMAIL', ''),
    'signed_audience'       => env('STACKKIT_CLOUD_TASKS_SIGNED_AUDIENCE', true),
    'handlers'              => [
      'default' => env('STACKKIT_CLOUD_TASKS_HANDLER'),
      // add your own handler urls here
    ],
],
```

The package or project that implements this cloud-tasks package is free to update the config as they desire and add other handlers.

With this config setup, the `taskHandlerUrl()` implementation of the job can fetch the appropriate handler url from the config if needed. Or add other logic to return the right handler url for that job. That's up to the job.

The main 'problem' I got with these hardcoded handlers: not all projects have "low performance" or "high performance". 
Without going into too much detail: my use case is that I have 2 different handlers: the default and one with a static outgoing IP address. This does not fit the hardcoded performance handlers.

Let me know what you think of this, or perhaps have other suggestion(s) based on this.